README.md

```diff
@@ -9,6 +9,38 @@ In Watchmen's `package.json` make sure to install [nightmareJS][nightmare].
 * [nightmare (@:octocat:)][nightmare]
 * [watchmen (@GH)](https://github.com/iloire/watchmen)
 
+### How To Use:
```

Comment:
while we're touching this file, how about changing line 8 to an NPM install line, like:
```
$ npm install watchmen-ping-nightmare --save
```
That seems to be the standard for node projects.